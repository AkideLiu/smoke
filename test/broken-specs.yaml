tests:
  - name: broken-specs
    args:
      - fixtures/broken-specs
    exit-status: 1
    stdout:
      - file: io/broken-specs.out-unix
      - file: io/broken-specs.out-windows
