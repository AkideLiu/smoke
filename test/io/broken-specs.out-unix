no-command
  There is no command.
no-inputs
  There are no args or STDIN values in the specification.
no-outputs
  There are no STDOUT or STDERR values in the specification.
missing-input-file
  The fixture "io/missing.in" does not exist.
missing-output-file
  The fixture "io/missing.out" does not exist.
missing-error-file
  The fixture "io/missing.err" does not exist.

6 tests, 6 failures
