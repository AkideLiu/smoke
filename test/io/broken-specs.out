no-command
  There is no command.
no-inputs
  There are no args or STDIN values in the specification.
no-outputs
  There are no STDOUT or STDERR values in the specification.
missing-input-file
  The fixture "fixtures/broken-specs/io/missing.in" does not exist.
missing-output-file
  The fixture "fixtures/broken-specs/io/missing.out" does not exist.
missing-error-file
  The fixture "fixtures/broken-specs/io/missing.err" does not exist.

6 tests, 6 failures
